test/new-e2e/pkg/environments/kubernetes.go

```diff
@@ -0,0 +1,26 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package environments
+
+import (
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+)
+
+// Kubernetes is an environment that contains a Kubernetes cluster, the Agent and a FakeIntake.
+type Kubernetes struct {
+	// Components
+	KubernetesCluster *components.KubernetesCluster
+	FakeIntake        *components.FakeIntake
+	Agent             *components.KubernetesAgent
+}
+
+var _ e2e.Initializable = &Kubernetes{}
+
+// Init initializes the environment
+func (e *Kubernetes) Init(_ e2e.Context) error {
```

Comment:
The environment does not require to implement `Initializable`, if there's nothing to do, you can just not implement it.

Reply:
I leave it here because I though I may need it in a future iteration. But indeed better remove it since it useless for the moment